oml/losses/arcface.py

```diff
@@ -0,0 +1,99 @@
+import warnings
+from typing import Any, Dict, List, Optional
+
+import numpy as np
+import torch
+from torch import nn
+from torch.nn import functional as F
+from torchvision.ops import MLP
+
+
+class ArcFaceLoss(nn.Module):
+    def __init__(
+        self,
+        in_features: int,
+        num_classes: int,
+        criterion: Optional[nn.Module] = None,
+        label2category: Optional[Dict[Any, Any]] = None,
+        label_smoothing: Optional[float] = None,
+        m: float = 0.5,
+        s: float = 64,
+    ):
+        super(ArcFaceLoss, self).__init__()
+
+        if label2category is not None and label_smoothing is None:
+            warnings.warn("Label smoothing in arcface head will not be used!")
```

Comment:
Maybe assert? Is there any reason to pass it?

Reply:
It was passed to `__init__` every time (even if we don't want to use LS), but not now.
Made an assert.